seerep_msgs/core/label_category.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SEEREP_CORE_MSGS_LABEL_CATEGORY_H_
+#define SEEREP_CORE_MSGS_LABEL_CATEGORY_H_
+
+#include <functional>
+
+namespace seerep_core_msgs
+{
+struct LabelCategory
+{
+  std::string category;
+  std::vector<std::string> labels;
+  std::vector<int> labelsIdDatumaro;
+  std::vector<std::string> instances;
+  std::vector<int> instancesIdDatumaro;
+  std::string datumaroJson;
```

Comment:
Why is this not a `std::vector<Label>` from `label.h`? This way, we would also not rely on an index based matching of related data.

Reply:
Ah, I figured out why, this way the automatic conversion of `std::vector<>` to `datasets` from HighFive can be leveraged. See here: https://github.com/agri-gaia/seerep/blob/dd983f9cfbdf433ef766cb08a16d7f7d7aaf40f3/seerep_hdf5/seerep_hdf5_core/src/hdf5_core_general.cpp#L156-L180
However, I still think it's not a good approach.

Reply:
I agree, but we have to copy the data in such a structure at one point anyway. Thus, we can do it directly when we copy the data the first time and avoid copying it a second time. Maybe there are better approaches, but for the time being it works.